### PR TITLE
Filters UI cleanup

### PR DESCRIPTION
**Pull request by ghost**
Corresponds to issue [291](https://github.com/Kitware/UPennContrast/issues/291)

### CODE REVIEW

**Comment by luciemac** [comment]
Can you rebase on master?

File: src/views/dataset/DatasetInfo.vue [hunk 1]
```diff
@@ -285,21 +285,21 @@ export default class DatasetInfo extends Vue {
...
       const config = await store.createConfiguration({
-        name: `default ${formatDate(date)}`,
-        description: "default configuration"
+        name: `${dataset.name} default configuration`,
+        description: "Default configuration"
       });
 
       if (config === null) {
-        throw new Error("config was null when it shouldn't be");
+        throw new Error("Config not set");
```
Comment (luciemac) [inline src/views/dataset/DatasetInfo.vue:300]: Config -> configuration

File: src/views/dataset/MultiSourceConfiguration.vue [hunk 2]
```diff
@@ -104,16 +105,46 @@ export default class NewDataset extends Vue {
   get items() {
     return this.dimensions
       .filter(dim => dim.size > 0)
-      .map((dim: any) => ({ ...dim, key: `${dim.id}_${dim.source}` }));
+      .map((dim: IDimension) => {
+        let value = "";
+        const metadataID = this.dimensionTometadataId(dim.id);
```
Comment (luciemac) [inline src/views/dataset/MultiSourceConfiguration.vue:110]: dimensionTometadataId -> dimensionToMetadataId

File: src/views/dataset/MultiSourceConfiguration.vue [hunk 3]
```diff
@@ -104,16 +105,46 @@ export default class NewDataset extends Vue {
...
+        switch (dim.source) {
+          case Sources.Filename:
+            if (this.collectedMetadata) {
+              const exampleValues = this.collectedMetadata.metadata[metadataID];
+              value = exampleValues.slice(0, 3).join(", ");
+              if (exampleValues.length > 3) {
+                value = String.prototype.concat(value, "...");
```
Comment (luciemac) [inline src/views/dataset/MultiSourceConfiguration.vue:117]: value = value.concat(value, "...") ?

File: src/views/dataset/MultiSourceConfiguration.vue [hunk 4]
```diff
@@ -250,17 +308,19 @@ export default class NewDataset extends Vue {
...
       const frames: number = tile.frames?.length || 1;
       this.maxFramesPerItem = Math.max(this.maxFramesPerItem, frames);
       if (tile.IndexStride) {
-        this.assignmentNames.forEach(dimension => {
-          const stride = tile.IndexStride[`Index${dimension}`];
-          if (stride && stride > 0) {
-            this.strides[dimension] = stride;
-            this.areStridesSetFromFile = true;
+        Object.keys(this.dimensionNames).forEach(
+          (dimension: string | number) => {
```
Comment (luciemac) [inline src/views/dataset/MultiSourceConfiguration.vue:316]: dimensionNames = { XY: "Positions", Z: "Z", T: "Time", C: "Channels" };

Object.keys(dimensionNames) returns an Array of strings, so I guess _dimension_ type should only be "string".

File: src/views/dataset/MultiSourceConfiguration.vue (hunk 3 above)
Comment (bruyeret) [reply src/views/dataset/MultiSourceConfiguration.vue:117]: This part of the code has been edited in another branch

File: src/views/dataset/MultiSourceConfiguration.vue (hunk 4 above)
Comment (bruyeret) [reply src/views/dataset/MultiSourceConfiguration.vue:316]: Already fixed in another branch

**Pull request merged**